Header header
uint64 obj_id
float32 fly_x
float32 fly_y
float32 laser_x
float32 laser_y
int32 laser_power
uint8 mode
